pkg/vintagewc/cluster.go

```diff
@@ -25,7 +25,15 @@ func (o Object) GetAppsNamespace() string {
 }
 
 func (o Object) AppConfigName(app string) string {
-	return app
+	if app == "observability-bundle" {
+		return o.GetObject().GetName() + "-" + app
+	} else {
+		return app
+	}
+}
+
+func (o Object) ObservabilityBundleConfigLabelName(config string) string {
+	return config
```

Comment:
Should this not have some kind of logic ?

Reply:
The method above (`AppConfigName`) was doing exactly the same before I updated it to match the needs for the WCs so I guess this is fine